modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefinePrefiltersAndSortsToRetrieveOnlyASelectionOfEntriesInThePreferredOrder/AllowUsersToDefineSorts.testcase

```diff
@@ -0,0 +1,299 @@
+@component-name = "portal-headless"
```

Comment:
@magjed4289 The folder name(name of epic ticket) seems a little long. For these kinds of epic tickets, are we going to use a shorter name, for example `[UI]AllowUsersToDefinePrefiltersAndSorts`? cc @Hijky

Reply:
Yeah, we can definitely shorten it to [UI]AllowUsersToDefinePrefiltersAndSorts, for the rest of the cases in the future we can think of some changes like operating wth LPS or ayn other idea. Let's see what the tendency with long names will be ;D